docs/user_guide/assessment.rst

```diff
@@ -418,11 +418,55 @@ to take advantage of the inherent plotting capabilities of
 .. literalinclude:: ../auto_examples/plot_quickstart.py
     :language: python
     :start-after: # Analyze metrics using MetricFrame
+    :end-before: # Customize plots with ylim
 
 .. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_001.png
     :target: auto_examples/plot_quickstart.html
     :align: center
 
+It is possible to customize the plots. Here are some common examples:
```

Comment:
Nitpick: as the following paragraph starts with a header, can we replace the colon : with a dot . ? 
```suggestion
It is possible to customize the plots. Here are some common examples.
```